user-guide/aragon-court/aragon-court-glossary.html

```diff
@@ -0,0 +1,124 @@
+</p>
+<p>
+	<strong>Arguments<br>
+	</strong>
+</p>
+<p>
+	Text, audio, video, and images put forth to make a case in favor of one of the possible dispute outcomes (Allow, Block, or Refuse to vote).
+</p>
+<p>
+	<strong>Dispute<br>
+	</strong>
+</p>
+<p>
+	When an Aragon Court subscriber believes that an Agreement they are party to has been violated and they decide to try to block the disputable action or proposal using Aragon Court.
+</p>
+<p>
+	<strong>Draft<br>
+	</strong>
+</p>
+<p>
+	The process of randomly selecting jurors for a new dispute round from the pool of active jurors.
+</p>
+<p>
+	<strong>Final ruling<br>
+	</strong>
+</p>
+<p>
+	<strong></strong>The final ruling of a dispute is one of either:
+</p>
+<ol>
+	<li>The ruling that a plurality of jurors in the current round have voted for, provided there has been no appeal or it is the final appeal round, or</li>
+	<li>The ruling proposed by an appealing party, provided that no one challenges the appeal.</li>
+</ol>
+<p>
+	<strong>Juror<br>
+	</strong>
+</p>
+<p>
+	A participant in the Aragon Court protocol who has locked and activated a specified minimum amount of ANJ, making them eligible to be drafted to resolve disputes brought before the court.
+</p>
+<p>
+	<strong>Period</strong>
+</p>
+<p>
+	Disputes are broken up into multiple periods, which are themselves each made up of a specific number of terms. The periods of a dispute are:
+</p>
+<ul>
+	<li>Argument submission (optional)</li>
+	<li>Vote commit</li>
+	<li>Vote reveal</li>
+	<li>Appeal</li>
+	<li>Appeal challenge</li>
+</ul>
+<p>
+	<strong>Preliminary ruling<br>
+	</strong>
+</p>
+<p>
+	The ruling selected by the plurality of jurors in the current dispute round, which may become the final ruling provided there is no appeal.
+</p>
+<p>
+	<strong>Rewards<br>
+	</strong>
+</p>
+<p>
+	Aragon Court offers financial rewards to users who successfully perform certain actions, such as drafting jurors, appealing preliminary rulings, and voting in favor of the final ruling of a dispute. Some rewards, such as dispute fees, ruling fees, and subscription fees, are only given to active jurors. Other rewards, such as appeal fees and maintenance fees, can be earned by anyone.
+</p>
+<p>
+	<strong>Round<br>
+	</strong>
+</p>
+<p>
+	<strong></strong>Aragon Court disputes are made up of at most four rounds of juror deliberation.<strong></strong> Disputes are escalated from one round to the next only if someone appeals the preliminary ruling and someone else challenges the appeal.<strong><br>
+	</strong>
+</p>
+<p>
+	<strong>Slash<br>
+	</strong>
+</p>
+<p>
+	When a juror rules incoherently i.e. against the final ruling and their locked ANJ is taken from them and redistributed to jurors who ruled in favor of the final ruling.
+</p>
+<p>
+	<strong>Term<br>
+	</strong>
+</p>
+<p>
+	The basic unit of time in Aragon Court. One term is equal to 8 hours.
+</p>
+<p>
+	<strong>Voting<br>
```

Comment:
Not opposed to this definition, but describing the mechanism feels out of place here (maybe better linked to a "schelling point"?). Perhaps we could just describe commit/reveal and plurality voting here?